scripts/build.sh

```diff
@@ -13,7 +13,7 @@ NUM_PROCS=1
 BUILD_PROTO=0
 BUILD_DIST=0
 BUILD_DIST_DEV=0
-PYTHON="3.7"
+PYTHON="3.8"
```

Comment:
Changing this value just changes the default value for the script. We still need to run this script for each version of Python we support.

Reply:
Sure but people who just want to build coreml from source can run this script without any modification (also work on intel etc.) Using 3.7 as base value break this script for m1 users (and a lot of people dont know how to search for stuff like this)